src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -209,17 +209,6 @@ public void SearchPatternIgnoreSubDirectories()
         #region PlatformSpecific
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.Windows)]
-        public void WindowsSearchPatternLongSegment()
-        {
-            // Create a path segment longer than the normal max of 255
```

Comment:
Again, on this one I don't care what it throws, just as long as it fails with some sort of IO exception.